h/static/styles/excerpt.scss

```diff
@@ -0,0 +1,31 @@
+@import "compass/css3/images";
+@import "variables";
```

Comment:
I'm guessing that's not actually a problem because `variables.scss` is just variable definitions, but you're absolutely right -- it's not necessary. I'll remove it.